.github/workflows/ci.yml

```diff
@@ -372,13 +414,42 @@ jobs:
             Remove-Item -Path "${{ env.CARGO_HOME }}/config.toml"  -Force
           }
 
+  tests_pass:
+    name: Tests Pass
+    runs-on: ubuntu-latest
+    needs:
+      - style
```

Comment:
I think we should always run `style`, as it contains things like our typos checks and checks for `todo!` comments.